### ISSUE TITLE
Rome won't upload all built frameworks

### ISSUE THREAD

**Issue by stevedaontuc** [tmspzz/Rome#234]
## Enhancement Suggestion / Bug Report 

    Add an option to detect all re-build downstream frameworks & upload them a long with missing frameworks.

**Steps which explain the enhancement or reproduce the bug**

   For now, when we're building missing frameworks by this script:
```shell
$ rome list --missing --platform ios | awk '{print $1}' | xargs -I {} carthage build "{}" --platform ios
```
  Carthage will re-build all downstream dependencies, Rome should upload all of them instead of missing frameworks.

**Current behavior**

    Rome only upload missing frameworks.

**Suggested behavior**

    Rome should upload all re-built downstream frameworks a long with missing frameworks.

**Why would the enhancement be useful to most users**

    Some re-built downstream frameworks are only valid with missing frameworks at the moment. 

**Rome version:** 

    v0.23.3.64
**OS and version:** 

    MacOS Catalina 10.15.7

**Comment by tmspzz** [comment]
You and just run upload at the end and it will upload all frameworks.

Rome does not keep state and it shouldn't. It only deals with the cache. It's up to the user to know the set of frameworks to upload/download.

**Comment by stevedaontuc** [comment]
Yes, we can do that but it isn't a nice way to do. I understand this optimization is not easy since Carthage won't export how many downstream dependencies would be re-built. Hope to see some expertises of Carthage here or I may need to post to Carthage page. Thank for your reply @tmspzz

**Comment by tmspzz** [comment]
You can parse Carthage's output after the line

`rebuilding all downstream dependencies`

And pipe the names of the frameworks that have been rebuilt to Rome
